imsv-docs-docusaurus/openapi/endpoints/login/auth-token-exchange.yaml

```diff
@@ -0,0 +1,49 @@
+post:
+  tags:
+    - authentication
+  summary: Auth Token Exchange
+  description: |
+    The token endpoint returns a new access token for a user without requiring the user to re-authenticate.
+  requestBody:
+    content:
+      application/json:
+        schema:
+          type: object
+          properties:
+            refreshToken:
+              type: string
+              description: The token previously issued to the client when login.
+              example: "cda232434dsfds342923frENnKthng3ji4.dseyJ1c2VyX2lkIjoidXNlcl8xMjM0NTYiLCJ0b2tlbl9pZCI6InRva2VuXzc4OTEwIiwiaWF0IjoxNjE1MjY5MTIyLCJleHAiOjE2MTU4NzM5MjJ9.Qy4f3hK6S9DqibtV78iqMzT0NSXx8OH8R3qL_UfzpM4km1ujGbnkIjUn"
+            authCode:
```

Comment:
i dont think you need authCode in this version of the docs yet. that is something @immersve/spoink will address in upcoming work